lib/relay.js

```diff
@@ -270,6 +276,7 @@ function responseHandler(filterRules, config, io) {
       }
 
       logger.debug(logContext, 'sending websocket request over HTTP connection');
+      headers.authorization = stringifyIfObject(headers.authorization);
```

Comment:
Why you need `stringifyIfObject` if you already doing `headers.authorization = {};`?🧐

Reply:
from https://github.com/snyk/broker/blob/fix/broker-auth/lib/relay.js#L206 onward, we MAY modify this object to be a string.

Reply:
ok. But what if you just do `headers.authorization = '{}';` in line [205](https://github.com/snyk/broker/pull/253/files#diff-3a518f5293da3cbae4467a305fb2a077R205)?
What all expected `headers.authorization` values?

Reply:
I'm not sure I follow, where are you saying we might modify this object to be a string @aviadhahami ? I'd expect the thing that's used to create the req object to be an object.

Reply:
see for example line 222
https://github.com/snyk/broker/blob/fix/broker-auth/lib/relay.js#L222

Reply:
I have a bunch of questions here around things like..
What else can be in this auth object? 
Could we not just set `headers.authorization` to an empty string rather than an object?
Do we know what the contents of result.auth looks like? Is it just a string? Could we just make it one?
Is any of this covered by tests?

Reply:
values - IMO it may contain string, null or an object
Tests cover String case for the auth header